issuance/issuance.go

```diff
@@ -76,13 +76,13 @@ type IssuerConfig struct {
 // Only one of File, ConfigFile, or PKCS11 should be set.
 type IssuerLoc struct {
 	// A file from which a private key will be read and parsed.
-	File string
+	File string `validate:"required_without=ConfigFile"`
 	// A file from which a pkcs11key.Config will be read and parsed, if File is not set.
-	ConfigFile string
+	ConfigFile string `validate:"required_without=PKCS11"`
 	// An in-memory pkcs11key.Config, which will be used if ConfigFile is not set.
-	PKCS11 *pkcs11key.Config
+	PKCS11 *pkcs11key.Config `validate:"required_without=ConfigFile"`
```

Comment:
All three of these should be `required_without_all=Other Two`. Any one of these three fields being set is sufficient for an IssuerLoc to load correctly.

And potentially they could also all be `excluded_with=The Ones Above Them`, to enforce the priority ordering of File > ConfigFile > PKCS11, but that would technically be a new constraint.